### PR TITLE
WE-165: Gatsby helmet meta tags

### PR DESCRIPTION
**Pull request by maxharrisnet**
- This installs Gatsby Helmet for handling meta data
- Adds tags to match those existing in production

### CODE REVIEW

File: gatsby-site/gatsby-config.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 module.exports = {
   siteMetadata: {
-    title: `Support Documents`,
-    siteUrl: `https://support.sparkpost.com`,
+    siteName: `SparkPost Support Documentation`,
+    url: `https://support.sparkpost.com`,
+    title: `SparkPost Support Documentation`,
+    defaultTitle: "Default title",
```
Comment (EvanYoung-SP) [inline gatsby-site/gatsby-config.js:6]: This is from prod? Let's give it something, at least. Maybe just the same as the title?

File: gatsby-site/src/components/seo.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    </Helmet>
+  );
+};
+
+export default SEO;
+
+SEO.propTypes = {
+  title: PropTypes.string,
+  description: PropTypes.string,
+  // image: PropTypes.string,
```
Comment (EvanYoung-SP) [inline gatsby-site/src/components/seo.js:69]: Are these comments being saved for later or can we remove them and the ones on line 76-77?

File: gatsby-site/gatsby-node.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const path = require("path");
```
Comment (EvanYoung-SP) [inline gatsby-site/gatsby-node.js:1]: Is there something that changed in this file? It looks the same, but it says the whole thing is new?

**Review by EvanYoung-SP — CHANGES_REQUESTED** [review]
A few comments that are pretty minor. The only major change that needs to be addressed is the build failure.

File: gatsby-site/gatsby-node.js (hunk 3 above)
Comment (maxharrisnet) [reply gatsby-site/gatsby-node.js:1]: I don't think I changed anything here. Weird that it is showing up as a new file. This was created when I installed Remark.

File: gatsby-site/gatsby-config.js (hunk 1 above)
Comment (maxharrisnet) [reply gatsby-site/gatsby-config.js]: Added some better values here, thanks!

**Review by EvanYoung-SP — APPROVED** [review]
I'm going to go ahead and approve the PR, but the build is still failing, so we'll need to have that resolved before we can merge.

**Pull request merged**